src/filter_coef.h

```diff
@@ -614,3 +614,389 @@ float quiskFilt120t480[480] = {
  0.000004115682735322, 0.000002322193058869, 0.000001038946634000, 0.000000197734700398, -0.000000267011791999,
 -0.000005050567303837 };
 
+// BPF generated from http://t-filter.engineerjs.com/ using the following:
+//
+//     # taps = 256
+//     0- 900 Hz: -60dB attenuation
+//  1000-2000 Hz: 0.1dB ripple
+//  2100-4000 Hz: -60dB attenuation
+//
+// Note that the coefficients are reversed. This is because after we add the current x[t] to the
+// lookback buffer, we start calculating from x[t - N] and loop back around, making b[0] * x[t] the
```

Comment:
I think the coeffs are symmetrical, so it might run just fine without reversal.  As per my comments I would prefer not to change coeffs at this time - it may affect the compressor/PAPR.